packages/wrangler/src/pages.tsx

```diff
@@ -1,13 +1,29 @@
 import type { BuilderCallback } from "yargs";
 import { join } from "path";
 import { tmpdir } from "os";
-import { existsSync, lstatSync, readFileSync } from "fs";
+import { closeSync, existsSync, lstatSync, openSync, readFileSync } from "fs";
+import type { ChildProcess } from "child_process";
 import { execSync, spawn } from "child_process";
 import { Headers, Request, Response } from "undici";
 import type { MiniflareOptions } from "miniflare";
 import type { RequestInfo, RequestInit } from "undici";
 import open from "open";
 import { watch } from "chokidar";
+import { buildWorker } from "../lib/functions/buildWorker";
+import type { Config } from "../lib/functions/routes";
+import { writeRoutesModule } from "../lib/functions/routes";
+import { generateConfigFromFileTree } from "../lib/functions/filepath-routing";
+
+const RUNNING_PROCESSES: ChildProcess[] = [];
+const EXIT = (message?: string) => {
```

Comment:
I would really appreciate it if you made all top level functions not be expressions; i.e - instead of `const X = (...) => ...`, please use `function X(...){...}`, thank you!